packages/clay-form/src/Checkbox.tsx

```diff
@@ -95,11 +95,11 @@ const ClayCheckbox = React.forwardRef<HTMLInputElement, IProps>(
 						{label && (
 							<span className="custom-control-label-text">
 								{label}
+
+								{children}
```

Comment:
I think this will break other use cases like Card, it doesn't need the label but still wants to render a children, we can maybe keep the `children` in place of before if the label is not defined.